worker/crates/action-sink/src/file/cesium3dtiles.rs

```diff
@@ -0,0 +1,380 @@
+    }
+    fn finish(&self, ctx: NodeContext) -> Result<(), BoxedError> {
+        let mut tree = TileTree::default();
+        for content in self.contents.lock().unwrap().drain(..) {
+            tree.add_content(content);
+        }
+
+        let tileset = cesiumtiles::tileset::Tileset {
+            asset: cesiumtiles::tileset::Asset {
+                version: "1.1".to_string(),
+                ..Default::default()
+            },
+            root: tree.into_tileset_root(),
+            geometric_error: 1e+100,
+            ..Default::default()
+        };
+
+        let gltf_json = serde_json::to_value(&tileset).unwrap();
+        let buf = gltf_json.to_string().as_bytes().to_owned();
+
+        let storage_resolver = Arc::clone(&ctx.storage_resolver);
+        let expr_engine = Arc::clone(&ctx.expr_engine);
+        let common_param = self.params.to_common_param();
+        let scope = expr_engine.new_scope();
+        let path = scope
+            .eval::<String>(common_param.output.as_ref())
+            .unwrap_or_else(|_| common_param.output.as_ref().to_string());
+        let output = Uri::from_str(path.as_str())?;
+
+        let storage = storage_resolver
+            .resolve(&output)
+            .map_err(crate::errors::SinkError::file_writer)?;
+
+        let root_tileset_path = output.path().join(Path::new("tileset.json"));
+        storage
+            .put_sync(root_tileset_path.as_path(), bytes::Bytes::from(buf))
+            .map_err(crate::errors::SinkError::file_writer)?;
+        Ok(())
+    }
+}
+
+fn handle_city_gml_geometry(
+    output: &Uri,
+    storage_resolver: Arc<StorageResolver>,
+    city_gml: geomotry_types::CityGmlGeometry,
+) -> Result<Arc<Mutex<std::vec::Vec<TileContent>>>, crate::errors::SinkError> {
+    let contents: Arc<Mutex<Vec<TileContent>>> = Default::default();
+
+    let features = city_gml.features.clone();
+    for feature in features {
+        match handle_feature(output, storage_resolver.clone(), &city_gml, feature) {
+            Ok(contens) => {
+                contents
+                    .lock()
+                    .unwrap()
+                    .extend(contens.lock().unwrap().iter().cloned());
+            }
+            Err(e) => {
+                return Err(crate::errors::SinkError::file_writer(format!(
+                    "Feature handle Error: {:?}",
+                    e
+                )))
+            }
+        }
+    }
+    Ok(contents)
+}
+
+fn handle_feature(
+    output: &Uri,
+    storage_resolver: Arc<StorageResolver>,
+    city_gml: &geomotry_types::CityGmlGeometry,
+    feature: geomotry_types::GeometryFeature,
+) -> Result<Arc<Mutex<std::vec::Vec<TileContent>>>, crate::errors::SinkError> {
+    let typename = feature.ty.name();
+    let contents: Arc<Mutex<Vec<TileContent>>> = Default::default();
+    let ellipsoid = nusamai_projection::ellipsoid::wgs84();
+    let tile_id_conv = TileIdMethod::Hilbert;
+
+    let min_zoom = 12;
+    let max_zoom = 18;
```

Comment:
Declare min_zoom: Option<i64>, max_zoom: Option<i64> as parameters in Cesium3dtilesWriterParam.
If there is no setting, we would like to have the default values set to 12 and 18.

Reply:
Add parameters and set default value